content/en/hub/tasks/administrative/system-alerts.md

```diff
@@ -26,3 +26,36 @@ is how often the alert appears after the alert has been triggered.
 The alerts are grouped into sections based on the alert type. For example,
 alerts that are related to pools appear in the alert section *Storage*.
 
+
+## Email
+
+An automatic script sends a nightly email to the root user account containing important information such as the health of the disks. Alert events are also emailed to the root user account. Problems with Scrub Tasks are reported separately in an email sent at 03:00AM.
+
+*Note*: S.M.A.R.T. reports are mailed separately to the address configured in that service.
+
+The administrator typically does not read email directly on the TrueNAS® system. Instead, these emails are usually sent to an external email address where they can be read more conveniently. It is important to configure the system so it can send these emails to the administrator’s remote email account so they are aware of problems or status changes.
```

Comment:
```suggestion
The administrator typically does not read emails directly on the TrueNAS® system. Instead, these emails are usually sent to an external email address where they can be read more conveniently. It is important to configure the system so it can send these emails to the administrator’s remote email account so they are aware of problems or status changes.
```